roles/network_plugin/calico/tasks/check.yml

```diff
@@ -44,19 +44,11 @@
   delegate_to: "{{ groups['kube_control_plane'][0] }}"
 
 - name: Get current calico cluster version
-  shell: "set -o pipefail && {{ bin_dir }}/calicoctl.sh version  | grep 'Cluster Version:' | awk '{ print $3}'"
+  shell: "set -o pipefail && {{ bin_dir }}/calicoctl.sh version  | grep 'Client Version:' | awk '{ print $3}'"
   args:
     executable: /bin/bash
   register: calico_version_on_server
-  async: 10
-  poll: 3
-  run_once: True
-  until: calico_version_on_server.stdout != 'unknown'
-  retries: 5
-  delay: "{{ retry_stagger | random + 3 }}"
-  delegate_to: "{{ groups['kube_control_plane'][0] }}"
```

Comment:
Lines 51-57 should be kept also, Why are they being removed?

Reply:
Thanks for your review.

The `run_once: True` and `delegate_to: "{{ groups['kube_control_plane'][0] }}"` is removed by mistake. I'll add them back in new commit.

But for the rest of lines removed, I don't think it should be kept.

In general, the `async` is for some long tasks, and there is no `async_status` run after that.

Looking forward to your reply.

Reply:
Thanks for your explanation. The changes in this version look very clean！

/lgtm